leobot_control/launch/teleop.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="joy_dev" default="/dev/input/js0" />
+  <arg name="joystick" default="true" />
```

Comment:
This argument is never used. Please make sure if it is True then joystick nodes are launched else no joystick nodes. Use conditionals in launch file http://wiki.ros.org/roslaunch/XML#if_and_unless_attributes